tools/checkstyle/checkStyle.xml

```diff
@@ -336,13 +336,13 @@
             <property name="format" value="(^[A-Z][0-9]?)$|([A-Z][a-zA-Z0-9]*[T]$)"/>
             <message key="name.invalidPattern" value="Class type name ''{0}'' must match pattern ''{1}''."/>
         </module>
-        
+
         <module name="PackageName">
-            <property name="format" value="^org\.apache\.seatunnel(\.[a-zA-Z][a-zA-Z0-9]*)*$"/>
+            <property name="format" value="^[a-z]+(\.[a-z][a-z0-9]{1,})*$"/>
```

Comment:
@leo65535 please take a look at this

Reply:
hi @garyelephant, 

When we write `SeaTunnelConfigParser`, it will refer to some classes in `config` project, but these classes don't maked public, so we can only put `SeaTunnelConfigParser` in `com.typesafe.config.impl` package.

So, we need to chane the checkstyle rule here.